src/server/capacity_unit_calculator.cpp

```diff
@@ -175,6 +179,27 @@ void capacity_unit_calculator::add_multi_get_cu(dsn::message_ex *req,
     _read_hotkey_collector->capture_hash_key(hash_key, key_count);
 }
 
+void capacity_unit_calculator::add_batch_get_cu(dsn::message_ex *req,
+                                                int32_t status,
+                                                const std::vector<::dsn::apps::full_data> &datas)
+{
+    int64_t data_size = 0;
+    for (const auto &data : datas) {
+        data_size += data.hash_key.size() + data.sort_key.size() + data.value.size();
+        _read_hotkey_collector->capture_hash_key(data.hash_key, 1);
+    }
+
+    _pfc_batch_get_bytes->add(data_size);
+    add_backup_request_bytes(req, data_size);
+
+    if (status != rocksdb::Status::kOk && status != rocksdb::Status::kNotFound &&
+        status != rocksdb::Status::kIncomplete && status != rocksdb::Status::kInvalidArgument) {
+        return;
```

Comment:
It's better to call `add_read_cu(1)` here.

Reply:
This billing standard is consistent with the public cloud and does not need to be increased by 1

Reply:
So why did you add read cu in `capacity_unit_caculator::add_get_cu` with the same situation?

Reply:
https://github.com/apache/incubator-pegasus/pull/897/files/17e2298c3b49e417e541f3e3a1b04b6d2e8b4575#diff-b42a4dca9b87da17a30778429845362e0af5fdc249f1969bef60b6a2efd5e0b9R144

Thate is `NOT_FOUND`, `NOT_FOUND` is always need add 1